lib/Conch/Route/User.pm

```diff
@@ -136,7 +136,7 @@ Unless otherwise noted, all routes require authentication.
 
 =back
 
-=head3 C<< POST /user/me/revoke?send_mail=<1|0>& login_only=<0|1> or ?api_only=<0|1> >>
+=head3 C<< POST /user/me/revoke?send_mail=<1|0>& login_only=<0|1> or &api_only=<0|1> >>
```

Comment:
See my comment above for the other endpoint that has a similar URL.

Reply:
that's the same endpoint :)

Reply:
D'oh. Not enough sleep this week.